core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -222,14 +218,131 @@ public static String RToSqlUdf(List<String> RExps, List<String> selectedColumns,
 
     String selectStr = "";
     for (String udf : udfs) {
-      String exp = newColToDef.containsKey(udf) ? String.format("%s as %s", newColToDef.get(udf), udf) : String.format(
-          "%s", udf);
+      String exp = newColToDef.containsKey(udf) ?
+          String.format("%s as %s", newColToDef.get(udf), udf) :
+          String.format("%s", udf);
       selectStr += (exp + ",");
     }
 
     return selectStr.substring(0, selectStr.length() - 1);
   }
 
+  /**
+   * Create new columns or overwrite existing ones
+   *
+   * @param newColumnNames       Array of new column names.
```

Comment:
Maybe no in interactive scenarios, but once they write it up in a script, they may want to pack them all into a single function invoke.
Is there any technical disadvantage with this approach though? We can add wrapper which takes single expression, similar to `transformUDF`.